### PR TITLE
small refactoring

### PR DESCRIPTION
**Pull request by 8M00M8**

### CODE REVIEW

File: TestAutomationSolution/Utils/Extensions.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace TestAutomationSolution.Utils
+{
+    public static class Extensions
+    {
+        private static TimeSpan timeout = TimeSpan.FromSeconds(Double.Parse(ConfigurationManager.AppSettings["explicitTimeout"]));
+
+        public static void WaitForToBeVisible(this IWebElement element, string errorMessage)
+        {
+            DefaultWait<IWebElement> wait = new DefaultWait<IWebElement>(element)
```
Comment (Ukrainis) [inline TestAutomationSolution/Utils/Extensions.cs:15]: Why not use usual WebDriverWait?

File: TestAutomationSolution/App.config [hunk 2]
```diff
@@ -3,43 +3,23 @@
...
       </dependentAssembly>
     </assemblyBinding>
   </runtime>
-  <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.2" />
-  </startup>
   <appSettings>
-    <!-- Service Bus specific app setings for messaging connections -->
-    <add key="ClientSettingsProvider.ServiceUri" value="" />
+    <add key="shopUrl" value="http://demoshop24.com" />
```
Comment (Ukrainis) [inline TestAutomationSolution/App.config]: It was planned to be one of the tasks, not? To add url. in app.config file.

File: TestAutomationSolution/Utils/Extensions.cs (hunk 1 above)
Comment (8M00M8) [reply TestAutomationSolution/Utils/Extensions.cs:15]: Webdriver is not needed in this case

File: TestAutomationSolution/App.config (hunk 2 above)
Comment (8M00M8) [reply TestAutomationSolution/App.config]: OK, I agree

**Pull request merged**